massdriver/resource_artifact_test.go

```diff
@@ -23,14 +23,45 @@ func TestAccMassdriverArtifactBasic(t *testing.T) {
 	})
 }
 
+func TestAccMassdriverArtifactRename(t *testing.T) {
```

Comment:
I agree with you there, this doesn't exercise the case we have. This is my first exploration into the SDK tests but I _think_ we could create the TF state w/ a helper or raw map and then get a better repro. Thoughts?

Reply:
TBH, I'm not sure. I don't know much about these acceptance tests and the docs around them aren't great. What I **do** know is they are intended to actually [run and create real resources](https://www.terraform.io/plugin/sdkv2/testing/acceptance-tests) (and they're supposed to torn down after), which in our case is a bit moot since all we're doing is sending SNS topics and we short-circuit that if the `MASSDRIVER_...` environment variables aren't set. So even our acceptance tests here aren't really valid since they aren't operating like they would in orchestration.  They basically just check if the code executes w/out a runtime failure, but they don't really test the logic.

We def need to come back to these and improve them. I don't know if we want to tackle that right now, especially since this change should be temporary (until IDs are generated in schemas)

Reply:
Sounds great to me all around. I'll yank the second test case.